### PR TITLE
use the timer netmiko send_command

### PR DESCRIPTION
**Pull request by yeled**
This alleviates the bizarre "long lines" problem I was having with the send_command_expect Netmiko function not being able to find `#` again.

It's a bit fire and forget.. 🤷‍♂

### CODE REVIEW

File: napalm_opengear/opengear.py [hunk 1]
```diff
@@ -33,17 +33,11 @@ def __init__(self, hostname, username, password, timeout=60, optional_args=None)
...
-                    output = self.device.send_command(cmd)
-                    if "% Invalid" not in output:
-                        break
-            else:
-                output = self.device.send_command(command)
-            return output
-        except (socket.error, EOFError) as e:
-            raise ConnectionException(str(e))
+        """Wrapper for Netmiko's send_command_timer method.
+           We use this because OpenGear prompt is weird.
```
Comment (mirceaulinic) [inline napalm_opengear/opengear.py]: LOL. I'm seeing in the above that it previously used to catch `% Invalid` outputs (although I haven't bumped into 'em yet). Does `send_command_timing` do this natively?

File: napalm_opengear/opengear.py [hunk 2]
```diff
@@ -123,7 +117,7 @@ def load_merge_candidate(self, filename=None, config=None):
 
         if filename is not None:
             with open(filename, 'r') as f:
-                candidate = f.readlines()
+                candidate = [line.strip() for line in f if line.strip()]  # strip blanks
```
Comment (mirceaulinic) [inline napalm_opengear/opengear.py]: :rabbit: :hole:

File: napalm_opengear/opengear.py (hunk 1 above)
Comment (yeled) [reply napalm_opengear/opengear.py]: No it doesn't. In fact I think I just stole it from https://github.com/napalm-automation-community/napalm-s350/blob/da258ffd991454cea0a0b50c2f0de4af6d0a090a/napalm_s350/s350.py#L119 so it never did anything. 🤦‍♂

Comment (yeled) [reply napalm_opengear/opengear.py]: we should fix that :)

**Pull request merged**